spec/lib/task-data/base-tasks/dell-racadm-set-bios-base-spec.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe(require('path').basename(__filename), function () {
+    var base = require('./base-task-data-spec');
+
+    base.before(function (context) {
+        context.taskdefinition = helper.require('/lib/task-data/base-tasks/dell-racadm-set-' +
+            'bios-base.js');
```

Comment:
Just style suggestion.
This kind of line break is a little not so good for me :), if it is just because the line is too long, you can move the total string to next line, like:
```javascript
context.taskdefinition = helper.require(
    '/lib/task-data/base-tasks/dell-racadm-set-bios-base.js'
);
```